Camera.gd

```diff
@@ -0,0 +1,26 @@
+extends Camera
+
+onready var _player: Player = get_tree().get_root().find_node("Level", true, false).get_node("Player")
+var _initialized := false
+var _velocity: Vector3
+const _BASE_ACCELERATION = 0.01
+const _FRICTION = 0.9
+const _THRESHOLD_DISTANCE = 2 # Maximum distance from the player at which the camera won't move
+
+func _process(delta: float) -> void:
```

Comment:
`_process` is called as many times as possible by the engine, but `player` only updates position in `_physics_process`, so we shouldn't be wasting resources on unnecessary calls

wadya think